CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+# Contributing to pgai
+
+We appreciate any help the community can provide to make pgai better!
+
+You can help in different ways:
+
+* Open an [issue](https://github.com/timescale/pgai/issues) with a
+  bug report, build issue, feature request, suggestion, etc.
+
+* Fork this repository and submit a pull request
+
+For any particular improvement you want to make, it can be beneficial to
+begin discussion on the GitHub issues page. This is the best place to
+discuss your proposed improvement (and its implementation) with the core
+development team.
+
+Before we accept any code contributions, pgai contributors need to
+sign the [Contributor License Agreement](https://cla-assistant.io/timescale/pgai) (CLA). By signing a CLA, we can
+ensure that the community is free and confident in its ability to use your
+contributions.
+
+## Getting and building Postgres AI
+
+Please follow our README for [instructions on installing from source](./README.md#installation).
```

Comment:
I think this section is unnecessary. If they are going to be contributing, they will need a development environment, which should be fully described in DEVELOPMENT.md as noted in the next section.